src/main/java/core/basesyntax/dao/ProductDaoImpl.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.FruitsStorage;
+import core.basesyntax.model.FruitTransaction;
+import java.util.Map;
+
+public class ProductDaoImpl implements ProductDao {
+    private static final String DEFAULT_INFORMATION = "fruit,quantity";
+    private static final String SEPARATOR = ",";
+
+    @Override
+    public void update(FruitTransaction fruitTransaction, int count) {
+        FruitsStorage.FRUIT_MAP.put(fruitTransaction.getFruit(), count);
+    }
+
+    @Override
+    public int getQuantityOf(FruitTransaction fruitTransaction) {
+        Integer quantity = FruitsStorage.FRUIT_MAP.get(fruitTransaction.getFruit());
+        if (quantity == null) {
+            return 0;
+        }
+        return quantity;
+    }
+
+    @Override
+    public String getAllData() {
```

Comment:
rename this method as `createReport()` and move to the class `ReportServiceImpl`. 
from the `Main` class call the `ReportServiceImpl.createReport()`
you can parametrize method by input data needed for creating the report, thist will help to create the report and will help to create unit tests in the future.